include/gpioMonitor.hpp

```diff
@@ -18,9 +18,9 @@ namespace vpd
 class GpioEventHandler
 {
   public:
-    GpioEventHandler() = delete;
-    ~GpioEventHandler() = delete;
-    GpioEventHandler(const GpioEventHandler&) = delete;
+    GpioEventHandler() = default;
```

Comment:
As this class needs args to be passed, should we block creating of object using empty constructor ?

Reply:
Done